src/UNL/UCBCN/Calendar.php

```diff
@@ -97,6 +98,10 @@ function links()
                      'uidlastupdated' => 'users:uid');
     }
 
+    public function manageHref() {
+        return "/manager/" . $this->shortname . "/manage/";
```

Comment:
Well I think there was just some confusion here. /shortname/manage is good but I think that since this page is under the manager project, it should definitely start with /manager/...otherwise we're inviting some confusion into the URLs. We could go with /manager/shortname/